docs/content/en/docs/concepts/architecture/deployment-flow.md

```diff
@@ -0,0 +1,174 @@
+and initiates the next phase.
+
+## Summary of deployment flow
+
+To view these events on your cluster, execute:
+
+```shell
+kubectl get events -n <namespace> . 
+```
+
+### Pre-deployment phase
+
+TODO: Add intro statement about this phase.
+
+```shell
+AppPreDeployTasks
+  AppPreDeployTasksStarted
+  AppPreDeployTasksSucceeded OR AppPreDeployTasksErrored
+```
+
+### Pre-deployment evaluation phase
+
+TODO: Add intro statement about this phase.
+
+```shell
+AppPreDeployEvaluations
+  AppPreDeployEvaluationsStarted
+  AppPreDeployEvaluationsSucceeded OR AppPreDeployEvaluationsErrored
+```
+
+### Deployment phase
+
+The `AppDeploy` phase basically covers
+the entire deployment and check phase of the workloads.
+The `KeptnApp` just observes whether
+all pre and post-deployment tasks/evaluation are successful
+and that the pods are deployed successfully.
+When all activities are successful,
+the `KeptnApp` issues the `AppDeploySucceeded` event
+and continues to the next phase.
+If any of these activities fail,
+the `KeptnApp` issues the `AppDeployErrored` event
+and terminates the deployment.
+
+```shell
+AppDeploy
+  AppDeployStarted
+  WorkloadPreDeployTasks
+    WorkloadPreDeployTasksStarted
+    WorkloadPreDeployTasksSucceeded OR WorkloadPreDeployTasksErrored 
+  WorkloadPreDeployEvaluations
+    WorkloadPreDeployEvaluationsStarted
+    WorkloadPreDeployEvaluationsSucceeded OR WorkloadPreDeployErrored
+  WorkloadDeploy
+    WorkloadDeployStarted
+    WorkloadDeploySucceeded OR WorkloadDeployErrored
+  WorkloadPostDeployTasks
+    WorkloadPostDeployTasksStarted
+    WorkloadPostDeployTasksSucceeded OR WorkloadPostDeployTasksErrored 
+  WorkloadPostDeployEvaluations
+    WorkloadPostDeployEvaluationsStarted
+    WorkloadPostDeployEvaluationsSucceeded OR WorkloadPostDeployEvaluationsErrored
+  AppDeploySucceeded OR AppDeployErrored
+  ```
+
+### Post-deployment phase
+
+TODO: Add intro statement about this phase.
+
+```shell
+AppPostDeployTasks
+  AppPostDeployTasksStarted
+  AppPostDeployTasksSucceeded OR AppPostDeployTasksErrored
+```
+
+### Post-deployment evaluation phase
+
+TODO: Add intro statement about this phase.
+
+```shell
+AppPostDeployEvaluations
+  AppPostDeployEvaluationsStarted
+  AppPostDeployEvaluationsSucceeded OR AppPostDeployEvaluationsErrored
+```
+
+### Completed phase
+
+```shell
+Completed
+```
+
+## Events that are not part of the deployment flow
+
+Additional phases/states exist,
+such as those that describe what happens when something fails.
+
+Whenever something in the system happens (we create a new resource, etc.)
+a Kubernetes event is generated.
+The following events are defined as part of the Keptn Lifecycle Toolkit
+but they are not part of the deployment flow.
+These include:
+
+```shell
+CreateEvaluation
+ReconcileEvaluation
+ReconcileTask
+CreateTask
+CreateApp
+CreateAppVersion
+CreateWorkload
+CreateWorkloadInstance
+Completed
+Deprecated
+WorkloadDeployReconcile
+WorkloadDeployReconcileErrored
```

Comment:
odubajDT commented:

Actually I was not able to find these two phases nor events in the system